crates/core/src/client/tests/mod.rs

```diff
@@ -207,12 +207,13 @@ async fn test_get_logs_from_bigger_than_current() {
     let logs = client.get_logs(filter).await.unwrap();
 
     // Then
-    assert!(logs.is_empty());
+    assert_eq!(logs, FilterChanges::Empty);
 }
 
 #[tokio::test]
 async fn test_get_logs_to_less_than_from() {
     // Given
+    // let fixtures = fixtures(vec![wrap_kakarot!(JsonRpcMethod::BlockNumber)]);
```

Comment:
can you remove this comment please?

Reply:
yes ser! my apologies for ignoring this!